### PR TITLE
Fixed 9 broken links and removed a link

### PR DESCRIPTION
**Pull request by slayeh17**
In the spreadsheet (https://docs.google.com/spreadsheets/d/10t0NSlxQVbLS7_uPEJIY6AmkFib-HHqqj_UvGovJL8Q/edit#gid=0) it is written to remove any Navendu Pottekkat related links so I removed his entire row.
And fixed 8 broken members link and the Layer5 community link.

Signed-off-by: Debak Roy <104914725+slayeh17@users.noreply.github.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c707ae69a2cf7e780cad281331ac4166efea3e29 at: https://62f7af3d0d26992976423241--layer5.netlify.app

File: src/collections/programs/hacktoberfest-2020/index.mdx [hunk 1]
```diff
@@ -87,7 +87,7 @@ import hacktoberfest2020 from "./hacktoberfest2021.png";
         <a href="https://docs.google.com/presentation/d/1Wc5ALdn-G3fADJ8I6nJlyOGhV2XHVprYPqDvkZ1MEqY/edit#" target="_blank">slides</a>, <a href="https://youtu.be/67iy2JEp4Ss" target="_blank">recording</a>
       </td>
       <td>
-        <a href="/community/members/vijay-cherukuri">Vijay Cherukuri</a>
+        <a href="https://layer5.io/community/members/vijay-cherukuri">Vijay Cherukuri</a>
```
Comment (Nikhil-Ladha) [inline src/collections/programs/hacktoberfest-2020/index.mdx:90]: I doubt if this is needed. The use of `/` before the path ensures that the path is from the root of the site.

File: src/collections/programs/hacktoberfest-2020/index.mdx [hunk 2]
```diff
@@ -96,61 +96,53 @@ import hacktoberfest2020 from "./hacktoberfest2021.png";
...
           href="https://youtu.be/pIFSI7xtwFs" target="_blank">recording</a></td>
-      <td><a href="/community/members/rodolfo-martinez-vega">Rodolfo Martinez Vega</a></td>
-    </tr>
-     <tr>
-      <td>Jan. 28th, 2021</td>
-      <td><i>Beginner's guide to contributing to Meshery and mesheryctl</i></td>
-      <td><a
-          href="https://docs.google.com/presentation/d/1oUzWQpFeFbpIs_sejtOPbF9J4nigj9ziEAzoWxmm6ig/edit" target="_blank">slides</a>, <a
-          href="https://youtu.be/hh_kFLZx3G4" target="_blank">recording</a></td>
-      <td><a href="/community/members/navendu-pottekkat">Navendu Pottekkat</a></td>
```
Comment (Nikhil-Ladha) [inline src/collections/programs/hacktoberfest-2020/index.mdx:139]: Nope, you shouldn't be deleting the section. Instead, just remove the hyperlink for Navendu's name. That's all

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
The only change that you should be doing in this PR is removing the hyperlink for Navendu's name and nothing else. All the other links are not broken, can you point me to the page where it is broken?

File: src/collections/programs/hacktoberfest-2020/index.mdx (hunk 1 above)
Comment (slayeh17) [reply src/collections/programs/hacktoberfest-2020/index.mdx:90]: I tried clicking each one of them but none of them worked. so I added "https://layer5.io"

File: src/collections/programs/hacktoberfest-2020/index.mdx (hunk 2 above)
Comment (slayeh17) [reply src/collections/programs/hacktoberfest-2020/index.mdx:139]: ok I'll fix this

**Comment by slayeh17** [comment]
> The only change that you should be doing in this PR is removing the hyperlink for Navendu's name and nothing else. All the other links are not broken, can you point me to the page where it is broken?

Yeah if you go here https://github.com/layer5io/layer5/blob/master/src/collections/programs/hacktoberfest-2020/index.mdx
and click on any presenters name none of them work

**Comment by Nikhil-Ladha** [comment]
> > The only change that you should be doing in this PR is removing the hyperlink for Navendu's name and nothing else. All the other links are not broken, can you point me to the page where it is broken?
> 
> Yeah if you go here https://github.com/layer5io/layer5/blob/master/src/collections/programs/hacktoberfest-2020/index.mdx and click on any presenters name none of them work also the Layer5 community link does not work

Yeah because that's the MDX file and not the hosted page :)
If you got to the path where the page is hosted, the links are working and that's how the links should be.

**Comment by Nikhil-Ladha** [comment]
Ref: https://layer5.io/programs/hacktoberfest

**Comment by slayeh17** [comment]
> Ref: https://layer5.io/programs/hacktoberfest

Oh ok sorry got it I'll make the necessary changes.

**Comment by l5io** [comment]
🚀 Preview for commit 250d2d8d301dff53fbd8f6d67830e8d6fe1fa3a1 at: https://62f928de98d7c60f9030be6c--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Thank you.

**Comment by slayeh17** [comment]
> Thank you.

Thank you for guiding me too.😊

**Pull request merged**